File: fib.rs
fn main(){
	for i in 1..=16 {
		print!("{:?}, ", fibonacci(i));
	}
	println!("...");
}

fn fibonacci(n: u8) -> u16 {
	let r = if n < 3 { 1 } else { fibonacci(n-1) + fibonacci(n-2) };
	r
}

File: fib.c
#include <stdio.h>

int fibonacci(int n){
	return n < 3 ? 1 : fibonacci(n-1) + fibonacci(n-2);
}

int main(){
	for (int i = 0; i < 17; i++){
		printf("%d, ", fibonacci(i));
	}
	printf("...\n");
}